enable-vcp-image/enable-vcp-scripts/manager_pod.sh

```diff
@@ -1,2 +1,150 @@
 #!/bin/bash
+source $(dirname "$0")/common_func.sh
 
+# read secret keys from volume /secret-volume/ and set values in an environment
+read_secret_keys
+
+# connect to vCenter using VCP username and password
+export GOVC_INSECURE=1
+export GOVC_URL='https://'$k8s_secret_vcp_username':'$k8s_secret_vcp_password'@'$k8s_secret_vc_ip':'$k8s_secret_vc_port'/sdk'
+error_message=$(govc ls 2>&1 >/dev/null)
+
+if [ $? -eq 1 ]; then
+    if [$error_message == "govc: ServerFaultCode: Cannot complete login due to an incorrect user name or password."]; then
+        echo "Failed to login to vCenter using VCP Username:" $k8s_secret_vcp_username " and VCP Password:" $k8s_secret_vcp_password
+        exit 1;
+    fi
+fi
+
+# if Administrator user is passed as VCP user, then skip all Operations for VCP user.
+if [ "$k8s_secret_vc_admin_username" != "$k8s_secret_vcp_username" ]; then
+    # connect to vCenter using VC Admin username and password
+    export GOVC_URL='https://'$k8s_secret_vc_admin_username':'$k8s_secret_vc_admin_password'@'$k8s_secret_vc_ip':'$k8s_secret_vc_port'/sdk'
+
+
+    # Verify if the Datacenter exists or not.
+    govc datacenter.info $k8s_secret_datacenter &> /dev/null
+    if [ $? -eq 0 ]; then
+        echo "[INFO] Verified Datacenter:" $k8s_secret_datacenter is present in the inventory.
+    else
+        echo "[ERROR] Unable to find Datacenter:" $k8s_secret_datacenter.
+        exit 1;
```

Comment:
Can you please use different exit code like exit 2 here and extit 3, 4 etc.. below.
So that someone else using this script can programatically detect why it failed.